alyf-de/banking
# fix: Update transaction unallocated amnt while adding PEs

Fixes https://github.com/alyf-de/banking/issues/85

- Due to transaction unallocated amnt not being updated, wrong SI values were being assigned in Payment Entries

**To Test:**
- Bank Transaction having deposit/unallocated amount 50
- Create two Sales Invoices with totals 20 and 40 each.
- While reconciling select these sales invoices against the transaction
- **What used to happen:** the second PE is made with 40 allocated to the SI
- **What is expected:** the second PE is made with 30 allocated to the SI (since the first invoice allocates 20 out of 50 in the Bank Transaction)

> Tests need some `before_test` hooks and setup. Will do it soon, but not in this PR
